source/geod24/Registry.d

```diff
@@ -15,8 +15,8 @@ import geod24.concurrency;
 /// Ditto
 public shared struct Registry
 {
-    private Tid[string] tidByName;
-    private string[][Tid] namesByTid;
+    private Object[string] objectByName;
+    private string[][Object] namesByObject;
```

Comment:
I believe, if you apply the suggestion to have a channel for the "listen" operation, you can have this typed (and same for the other `Object` types in `localrest`